src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2488,27 +2488,31 @@ def aks_mesh_get_mesh_config(cmd, client, resource_group_name, name):
     from azure.cli.core import __version__ as local_version
     from packaging.version import parse
     from pprint import pprint
-    from kubernetes.client import CustomObjectsApi
     
-    kubernetes.config.load_kube_config()
+    print('---')
+    contexts, active_context = kubernetes.config.list_kube_config_contexts()
+    if not contexts:
+        print("No context in the kube config file.")
+        return  
```

Comment:
you should also raise/return an error here. You can look at the other error examples in this file. They import error handling from several packages, for example `azure.cli.core.azclierror` or `azure.core.exceptions`. You can check these out to see what error is the most applicable. Some of them will also let you pass in your error message and print it out for you